images/kubernetesonarm/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM larmog/armhf-alpine-java:jdk-8u73
+
+EXPOSE 9200 9300
```

Comment:
Yes, but they don't have to be defined in the Dockerfile for docker or Kubernetes to open the ports. Port 9200 and 9300 will be exposed without this layer.